source/Api.cpp

```diff
@@ -27,6 +27,14 @@ namespace Aws
     {
         Allocator *g_allocator = Aws::Crt::DefaultAllocator();
 
+        static struct aws_log_subject_info s_crt_cpp_log_subject_infos[] = {
+            {AWS_LS_CRT_CPP_GENERAL, "CrtCppGeneral", "Subject for aws-crt-cpp logging that defies categorization."},
+            {AWS_LS_CRT_CPP_CANARY, "CrtCppCanary", "Subject for aws-crt-cpp canary logging."}};
```

Comment:
Shouldn't this be in the canary, via a new package ID?

Reply:
I was thinking because it exists as part of the C++ CRT code base, it was okay here.  But yeah, I agree, it's probably cleaner inside of the canary itself given it's a separate executable.